util/src/main/java/net/automatalib/util/partitionrefinement/PaigeTarjanInitializers.java

```diff
@@ -410,4 +374,25 @@ public static void initDeterministic(PaigeTarjan pt,
         pt.removeEmptyBlocks();
     }
 
+    private static Block determineBlockForClassification(
```

Comment:
Maybe something like `getOrCreateSuccBlock` would be a more fitting name since the method also mutates the datastructure via `createBlock`.